yelp_package/trusty/Dockerfile

```diff
@@ -24,20 +24,14 @@ RUN apt-get -q update && \
         libssl-dev \
         libyaml-dev \
         python3.6-dev \
+        python-pip \
+        python-tox \
         wget \
     && apt-get -q clean
 
-RUN wget https://bootstrap.pypa.io/get-pip.py -O /tmp/get-pip.py
-RUN python3.6 /tmp/get-pip.py
-RUN pip3.6 install -U tox wheel setuptools PyYAML
-
 RUN cd /tmp && \
     wget http://mirrors.kernel.org/ubuntu/pool/universe/d/dh-virtualenv/dh-virtualenv_1.0-1_all.deb && \
     gdebi -n dh-virtualenv*.deb && \
     rm dh-virtualenv_*.deb
```

Comment:
Probably could install dh-virtualenv via apt too?

Reply:
I could, but it is a super old version